google/cloud/pubsublite/options.h

```diff
@@ -63,6 +63,15 @@ struct PublishFlushAlarmPeriodOption {
   using Type = std::chrono::milliseconds;
 };
 
+/**
+ * Whether idempotence is enabled, where the server will ensure that unique
```

Comment:
The first line in these comments is the "brief" description and ideally should fit in a single line.  Consider rephrasing.  You may also want to reference the cloud site docs, e.g.:

```cc
  * 
  * @see https://cloud.google.com/pubsub/lite/docs/publishing#publish_idempotency
  */
```